internal/system/agent/executor/os.go

```diff
@@ -43,19 +44,19 @@ func findAndStopProcess(output string, err error, process string) error {
 				tokens := strings.Split(line, " ")
 				pid, err = strconv.Atoi(tokens[0])
 				if err != nil {
-					log.Fatalf("Error converting PID to integer: " + err.Error())
+					log.Printf("Error converting PID to integer: %s" + err.Error())
```

Comment:
This line should be

`log.Printf("Error converting PID to integer: %s" , err.Error())`